docs-website/sidebars.js

```diff
@@ -247,6 +242,20 @@ module.exports = {
             },
           ],
         },
+        {
+          Slack: [
+            {
+              type: "doc",
+              id: "docs/managed-datahub/slack/saas-slack-setup",
+              className: "saasOnly",
+            },
+            {
+              type: "doc",
+              id: "docs/managed-datahub/slack/saas-slack-app",
```

Comment:
Not really sure "Slack" or "Slack App" is the best name for these docs.. maybe Slack Integration /  Slack App Features? 
I don't have a good idea for this so I'll defer to you  😂 

<img width="310" alt="image" src="https://github.com/user-attachments/assets/a4ff470c-c513-4f07-9e41-868c1487464e">